pkg/api/kubegraph/nodes/types.go

```diff
@@ -234,3 +242,28 @@ func (n SecretNode) String() string {
 func (*SecretNode) Kind() string {
 	return SecretNodeKind
 }
+
+func HorizontalPodAutoscalerNodeName(o *extensions.HorizontalPodAutoscaler) osgraph.UniqueName {
+	return osgraph.GetUniqueRuntimeObjectNodeName(HorizontalPodAutoscalerNodeKind, o)
+}
+
+type HorizontalPodAutoscalerNode struct {
+	osgraph.Node
+	*extensions.HorizontalPodAutoscaler
```

Comment:
This is the prevailing approach in this file, though. If we want to refactor all of them to not embed and get rid of `Object()` I'm all for it but let's do that as a follow-up.